### ISSUE TITLE
[java] CommentDefaultAccessModifierRule ignores constructors

### ISSUE THREAD

**Issue by jsotuyod** [pmd/pmd#536]
**Rule Set:**
java-comments/CommentDefaultAccessModifierRule

**Description:**
The rule detects uncommented package-private fields, methods and inner classes, but fails to detect constructors.

**Code Sample demonstrating the issue:**

```java
public class Foo {
    Foo() { } // warning expected here
}

public enum Bar {
    ONE, TWO;

    Bar() { } // but no warning expected here (implicitly private)
}
```

**Running PMD through:** *Any*
